README.rst

```diff
@@ -0,0 +1,591 @@
+tractor
+=======
+A minimalist `actor model`_ built on multiprocessing_ and trio_.
+
+|travis|
+
+.. |travis| image:: https://img.shields.io/travis/tgoodlet/tractor/master.svg
+    :target: https://travis-ci.org/tgoodlet/tractor
+
+``tractor`` is an attempt to take trionic_ concurrency concepts and apply
+them to distributed-multicore Python.
+
+``tractor`` lets you run and spawn Python *actors*: separate processes which are internally
+running a ``trio`` scheduler and task tree (also known as an `async sandwich`_).
+
+Actors communicate with each other by sending *messages* over channels_, but the details of this
+in ``tractor`` is by default hidden and *actors* can instead easily invoke remote asynchronous
+functions using *portals*.
```

Comment:
I mean, this is a semantic game, but arguable function calls in python *are* messages when you consider how attribute access works in Python. Its not really all that hidden.

You could clean this up by saying "sending messages over channels to easily invoke remote ...."

Reply:
Agreed, I wanted to change this sentence anyway.